R/MsBackendTimsTof.R

```diff
@@ -137,7 +139,8 @@ setClass("MsBackendTimsTof",
                                                                   "file"))),
                                fileNames = integer(),
                                readonly = TRUE,
-                               version = "0.1"))
+                               version = as.character(
```

Comment:
honestly, I'm not totally convinced by this. I know, it would allow us to keep track of the version of the package with which a class was created - but it would not reflect the *version* of the *object*. The version of the object should just change if its definition changes, i.e. if slots were added or removed. I would suggest to keep the old version, but am open to discuss.

Reply:
Now I see what you mean: I thought `version` was meant to be related to the package versioning, not the object _structure_ itself. I see this slot is inherited from the virtual class MsBackend, so it makes sense to keep its definition consistent 

So yes, let's keep the old version then 👍